### DISCUSSION TITLE
Alternative to dnsvalidator?

### DISCUSSION THREAD

**Discussion by IOCsec** [projectdiscovery/shuffledns#224] [Q&A]
In the shufledns description under Prerequisites specify "The tool also needs a list of valid solvers. The [dnsvalidator](https://github.com/vortexau/dnsvalidator) project can be used to generate these lists."

You should recommend another project, this is because dnsvalidator looks like an abandoned project, no updates in the last 2 years.
In addition dnsvalidator for all those using newer versions of python3 (3.8+) will give error when running:
```
dnsvalidator -m -tL https://public-dns.info/nameservers.txt -threads 20 -o resolvers.txt
Traceback (most recent call last):
  File "~/dnsvalidator/dnsvalidator/dnsvalidator.py", line 15, in <module>
    from .lib.core.input import InputParser, InputHelper
ImportError: attempted relative import with no known parent package
```
And the solution of running the script with `-m` does not work.

Please consider presenting another project as an alternative to dnsvalidator.

**Comment by ehsandeep** [comment]
https://github.com/trickest/resolvers is another option.
